### DISCUSSION TITLE
Schedule question

### DISCUSSION THREAD

**Discussion by 1mfaasj** [upptime/upptime#984] [Q&A]
Hi
Upptime checks the configured sites every 5 minutes right?
But on the response time page it only updates once a day, why is this?

Which of the schedules does change that?
https://upptime.js.org/docs/configuration/#ci-schedule

Is there a reason why not all schedules are set to 5 minutes?

Thanks

**Comment by AnandChowdhary** [comment]
Yes, once per day (by default, you can configure this) we update the response time data, but much more often (e.g., every 5 min) we check if the website us up.
